packages/ui/src/components/ContentRenderer/components/RichText/Link.tsx

```diff
@@ -8,18 +7,12 @@ interface Props {
 }
 
 export function Link({ node, children }: Props) {
-    const { track } = useAnalytics();
     const { href } = node;
 
-    function handleClick() {
-        track(STORY_LINK.CLICK, { href });
-    }
```

Comment:
Apparently, we don't track this even on the Link level anymore, as there is no constant for it in the latest `@prezly/analytics` package. 

We only track outbound links (leading to other websites) from [the global listener](https://github.com/prezly/analytics/blob/d7cc7cd20829f4e1da9321b4f3822c712ea8e62a/packages/analytics-nextjs/src/components/Analytics.tsx#L20-L40) defined in the Analytics package.